src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/index.jelly

```diff
@@ -50,13 +51,7 @@ THE SOFTWARE.
                                     <j:set var="it" value="${item}"/>
                                     <j:set var="instance" value="${null}"/>
                                     <j:set var="help" value="${descriptor.helpFile}"/>
-                                    <j:if test="${help != null}">
-                                        <tr>
-                                            <td colspan="3"/>
-                                            <f:helpLink url="${help}"/>
-                                        </tr>
-                                        <f:helpArea/>
-                                    </j:if>
+                                    <f:entry title="STEP: &lt;code&gt;${quasiDescriptor.symbol}&lt;code&gt;" help="${help}" />
```

Comment:
Instead of doing some magic to position the help for a step, we could add an entry and use it to anchor the help for the step.  
I want this to look like "STEP: `symbol`" but it seems quasiDescriptor.symbol} is not available when the block is expanded. 
@timja  @fqueiruga  do you know the magic to get the value down in the block?

Reply:
I suppose I could use this instead, keep things simple. 
```suggestion
                                    <f:entry title="Step parameters" help="${help}" />
```